src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -220,6 +213,7 @@ public static void setJDBCLoggingDirectory() {
 
   /**
    * validate whether proxy settings in the config is valid
+   * note: nonproxyhosts have no requirements, so they are not validated here
```

Comment:
Just a single comment (nit): Given the env-vars use `,` as delimiters while JVMs use `|` perhaps it's worth validating that `,` characters are not passed in this value? It could help avoid the troubleshooting required when a user copy-pastes things and does not carefully follow a document.